include/services/hid.hpp

```diff
@@ -32,6 +33,83 @@ namespace HID::Keys {
 		CirclePadUp = 1 << 30,     // Y >= 41
 		CirclePadDown = 1u << 31   // Y <= -41
 	};
+
+	constexpr const char* keyToName(u32 key) {
+		switch (key) {
+			case HID::Keys::A:
+				return "A";
+			case HID::Keys::B:
+				return "B";
+			case HID::Keys::Select:
+				return "Select";
+			case HID::Keys::Start:
+				return "Start";
+			case HID::Keys::Right:
+				return "Right";
+			case HID::Keys::Left:
+				return "Left";
+			case HID::Keys::Up:
+				return "Up";
+			case HID::Keys::Down:
+				return "Down";
+			case HID::Keys::R:
+				return "R";
+			case HID::Keys::L:
+				return "L";
+			case HID::Keys::X:
+				return "X";
+			case HID::Keys::Y:
+				return "Y";
+			case HID::Keys::CirclePadRight:
+				return "CirclePad Right";
+			case HID::Keys::CirclePadLeft:
+				return "CirclePad Left";
+			case HID::Keys::CirclePadUp:
+				return "CirclePad Up";
+			case HID::Keys::CirclePadDown:
+				return "CirclePad Down";
+			default:
+				return "Unknown key";
+		}
+	}
+
+	constexpr u32 nameToKey(const std::string& name) {
```

Comment:
Ditto
Also marking it constexpr might not be ideal since compilers have had problems with std::string in constexpr contexts until recently

Reply:
Also this isn't case-insensitive, should probably make it case-insensitive like all other similar configurations